bin/generate-apidocs.js

```diff
@@ -0,0 +1,8 @@
+
+'use strict';
+
+// Call sdocs
+require('child_process').spawn(
+  'sdocs', // Typically '/usr/local/bin/node'
```

Comment:
@mpatsute This line is failing with `ENOENT` because it can't find this command.